src/dashboards/utils/cellGetters.ts

```diff
@@ -34,7 +45,8 @@ export const isCellUntitled = (cellName: string): boolean => {
 export const getNewDashboardCell = (
   state: AppState,
   dashboard: Dashboard,
-  clonedCell?: Cell
+  clonedCell: Cell,
```

Comment:
for anyone that's curious this function only looks like it's ever being used in one place